Chapters 21 - 23

CodeWarrior-Hawaii opened the pull request:
OK, this is a big 'un.  I had forgotten that Chapter 22 was 42 million words long.  I have commits broken in to Punctuation, Callsigns, and Re-Linerating (undoing the forced word-wrapping).

Let me know what I missed because I am sure that I missed or screwed up something in this gigantic "Chapter".

CodeWarrior-Hawaii committed:
Fixes #31

CodeWarrior-Hawaii committed:
Fixes #32 - Punctuation for Part 1

CodeWarrior-Hawaii committed:
Fixes #32 - Callsigns for Part 1

CodeWarrior-Hawaii committed:
Fixes #32 - Puctuation for Part 2

CodeWarrior-Hawaii committed:
Fixes #32 - Callsigns for Part 2

CodeWarrior-Hawaii committed:
Whoops

CodeWarrior-Hawaii committed:
Fixes #32 - Punctuation for Part 3

CodeWarrior-Hawaii committed:
Fixes #32 - Callsigns for Part 3

CodeWarrior-Hawaii committed:
Fixes #32 - Punctuation for Part 4

CodeWarrior-Hawaii committed:
Fixes #32 - Callsigns for Part 4

CodeWarrior-Hawaii committed:
Fixes #32 - Punctuation for Part 5

CodeWarrior-Hawaii committed:
Fixes #32 - Callsigns for Part 5

CodeWarrior-Hawaii committed:
Fixes #32 - Punctuation for Part 6

CodeWarrior-Hawaii committed:
Fixes #32 - Callsigns for Part 6

CodeWarrior-Hawaii committed:
Fixes #32 - Punctuation Chapter 23

CodeWarrior-Hawaii committed:
Fixes #33 - Callsigns for Chapter 23

CodeWarrior-Hawaii committed:
Fixes #32 - The great Re-Linerating Chapter 22

CodeWarrior-Hawaii committed:
Fixes #31 - Relinerating for Chapter 21

ctwelve commented:
Oh Goddamn this is a biggie!

ctwelve reviewed (changes requested):
Hello. In the datelines you've deleted all trailing space. This is incorrect, because the (at least) two extra spaces after the dateline tell Markdown to do a linebreak without a new paragraph. Basically works like `<br>` does in HTML.

Please put them back, because @HamboneHFY considers the stacked datelines essential to the presentation.

ctwelve commented on content/books/deathworlders/chapter-22-warhorse-pt1-first-year.md:
```diff
@@ -1,928 +1,566 @@
----
-
-**Date Point: 4y 10m AV**  
+--- title: "Chapter 22: Warhorse Part 1—The First Year" date: 2015-09-18 aliases:   - /book/deathworlders/chapter-22-warhorse ---
```
The metadata block is...pretty thoroughly broken, in all the files. It's a YAML front matter block encased in three dashes. It has to be formatted **precisely**  and will not tolerate this.

ctwelve commented on content/books/deathworlders/chapter-22-warhorse-pt2-second-year.md:
```diff
@@ -1,366 +1,233 @@
----
-
-**Date Point: First Contact Day, 6y AV**  
+--- title: "Chapter 22: Warhorse Part 2—The Second Year" date: 2015-09-19 aliases:   - /book/deathworlders/chapter-22-warhorse/second-year ---
```
Same deal here. Broken metadata! And...so on through all the files.

ctwelve reviewed (changes requested):
Metadata! Also I note the ellipsis character has appeared; we don't want it! Please replace with three periods everywhere.

Commit:
Merge branch 'master' into Chapter_21-23

CodeWarrior-Hawaii committed:
Fixes #32 - Unfuxxored headers, and section dates.  Also some spelling fixes for NOVA HOUND

CodeWarrior-Hawaii committed:
Merge branch 'Chapter_21-23' of https://github.com/deathworlders/online into Chapter_21-23

CodeWarrior-Hawaii committed:
Removal of ALL ELLIPSES

Let this be a lesson to me... We want three periods, not the damned ellipsis character.

CodeWarrior-Hawaii commented:
Alright.  Ellipsis character burninated with extreme prejudice.  Chapter metadata blocks fixed.  Spaces reverted to section headers.  Also some spelling and punctuation fixes.

CodeWarrior-Hawaii committed:
Merge branch 'master' into Chapter_21-23

ctwelve reviewed (approved):
Looks good!
